### ISSUE TITLE
Do not add units to css custom properties

### ISSUE THREAD

**Issue by TrySound** [facebook/react#9963]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

```js
<div style={{ '--value': 5 }}></div>
```
becomes
```html
<div style="--value: 5px;"></div>
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: https://jsfiddle.net/84v837e9/).**

https://codepen.io/TrySound/pen/MojVOp

**What is the expected behavior?**

Leave values same as they are passed.

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**

React 15.6.0

**Comment by gaearon** [comment]
Would you like to send a fix? We still have some time before cutting 15.6.1.

**Comment by TrySound** [comment]
Sure.

**Comment by gaearon** [comment]
We're looking to cut it at most tomorrow so please let us know if you can't finish it today, and we'll look into it.

**Comment by gaearon** [comment]
Thanks!

**Comment by gaearon** [comment]
Should be fixed in 15.6.1. Please verify.
